reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md

```diff
@@ -198,75 +262,29 @@ LastAccessTime
-TypeName: System.ServiceProcess.ServiceController
-Name                      MemberType    Definition
-----                      ----------    ----------
-Name                      AliasProperty Name = ServiceName
-RequiredServices          AliasProperty RequiredServices = ServicesDependedOn
-Disposed                  Event         System.EventHandler Disposed(System.Object, System.EventArgs
-Close                     Method        System.Void Close()
-Continue                  Method        System.Void Continue()
-CreateObjRef              Method        System.Runtime.Remoting.ObjRef CreateObjRef(type requestedTy
-Dispose                   Method        System.Void Dispose()
-... 
-PS> Get-Member -InputObject $S
-TypeName: System.Object[]
-Name           MemberType    Definition
-----           ----------    ----------
-Count          AliasProperty Count = Length
-Address        Method        System.Object&, mscorlib, Version=2.0.0.0, Cultu
-Clone          Method        System.Object Clone()
-CopyTo         Method        System.Void CopyTo(array array, int index), Syst
-Equals         Method        bool Equals(System.Object obj)
-Get            Method        System.Object Get(int )
-GetEnumerator  Method        System.Collections.IEnumerator GetEnumerator()
-GetHashCode    Method        int GetHashCode()
-...
+Attributes
 ```
 
-This example shows how the *InputObject* parameter works in the **Get-Member** cmdlet.
-When you use the *InputObject* parameter to get the members of a collection, **Get-Member** gets the members of the collection.
-When you pipe a collection of objects to **Get-Member**, **Get-Member** gets the members of each item in the collection.
-
-The first command gets the services on the local computer and saves the services in the $S variable.
-
-The second command pipes the $S variable to the **Get-Member** cmdlet.
-**Get-Member** gets the types of each member of $S and the members of that type.
-In this case, it gets ServiceController objects and lists the members, such as RequiredServices and Close.
-
-The third command uses the *InputObject* parameter of **Get-Member** to submit the $S variable.
-**Get-Member** gets the type (**System.Object\[\]**) and the members of the collection (or array) of **ServiceController** objects, such as Count and Clone.
-
 ## PARAMETERS
 
 ### -Force
-Adds the intrinsic members (PSBase, PSAdapted, PSObject, PSTypeNames) and the compiler-generated get_ and set_ methods to the display.
-By default, **Get-Member** gets these properties in all views other than Base and Adapted, but it does not display them.
 
-The following list describes the properties that are added when you use the *Force* parameter:
+Adds the intrinsic members and the compiler-generated **get_** and **set_** methods to the display.
+The following list describes the properties that are added when you use the **Force** parameter:
+
+- PSBase: The original properties of the .NET Framework object without extension or adaptation.
```

Comment:
I'd consider making each property bold in the list to help them stand out.

Reply:
Fixed